freezegun/api.py

```diff
@@ -20,12 +21,17 @@
 except ImportError:
     import copyreg
 
+
 # Stolen from six
 def with_metaclass(meta, *bases):
     """Create a base class with a metaclass."""
     return meta("NewBase", bases, {})
 
 
+def _is_cpython():
+    return platform.python_implementation() == "CPython"
```

Comment:
On the old `ctypes` branch, @spulec has the cpython test checking for whether `platform` has even the `python_implementation` attribute - see https://github.com/spulec/freezegun/commit/f429b5ae015a11c2cd25feb2ce06f868aad226de#diff-aa9b211ef57daf3482592c4cc9180558R23 . I don't know in what python interpreters this is the case, but I guess best to act defensively as I don't think this would just be added randomly.